tools/license-builder.sh

```diff
@@ -33,7 +33,7 @@ addlicense "c-ares" "deps/cares" \
            "$(sed -e '/^ \*\/$/,$d' -e '/^$/d' -e 's/^[/ ]\* *//' ${rootdir}/deps/cares/src/ares_init.c)"
 addlicense "HTTP Parser" "deps/http_parser" "$(cat deps/http_parser/LICENSE-MIT)"
 addlicense "ICU" "deps/icu" \
-           "$(sed -e '1,/COPYRIGHT AND PERMISSION NOTICE/d' -e '/^<hr/,$d' -e 's/^<\/*p>$//' ${rootdir}/deps/icu/license.html)"
+           "$(sed -e '1,/ICU License - ICU 1\.8\.1 and later/d' -e :a -e 's/<[^>]*>//g;/</N;//ba' ${rootdir}/deps/icu/license.html)"
```

Comment:
@srl295 license-builder.sh was introduced in 031b87d42d and I don't believe we've had a release since then. The idea is to touch LICENSE as little as possible and have a tool that automatically builds it and keeps it clean, so it overwrote your previous work (sorry, but it was too manual!).

Just let us know when we get LICENSE.txt and we can update it here, or you could PR it yourself, when you do away with the HTML it'll be much easier!